pkg/installmanager/installmanager.go

```diff
@@ -267,8 +266,13 @@ func (m *InstallManager) Run() error {
 	}
 	m.log.Infof("copied %s to %s", m.InstallConfigMountPath, destInstallConfigPath)
 
-	if knownHost, ok := cd.Annotations[hiveSSHKnownHostsAnnotation]; ok {
-		err = m.setupSSHKnownHost(knownHost)
+	if cd.Spec.Provisioning != nil && len(cd.Spec.Provisioning.SSHKnownHosts) > 0 {
+		err = m.setupSSHUser()
+		if err != nil {
+			m.log.WithError(err).Error("error setting up SSH known_hosts")
+			return err
+		}
+		err = m.writeSSHKnownHosts("/home/hive", cd.Spec.Provisioning.SSHKnownHosts)
```

Comment:
When we were working on the libvirt integration we ended up needing to change the home directory of the user so it pointed to a shared volume. Should this read the HOME environment variable instead of assuming "/home/hive"?

Reply:
Good idea, adjusting...